cmd/kubeadm/app/apis/kubeadm/v1beta4/defaults.go

```diff
@@ -99,6 +102,10 @@ func SetDefaults_ClusterConfiguration(obj *ClusterConfiguration) {
 		obj.ClusterName = DefaultClusterName
 	}
 
+	if obj.EncryptionAlgorithm == "" {
+		obj.EncryptionAlgorithm = DefaultEncryptionAlgorithm
+	}
+
```

Comment:
not necessary in this pr, we don't have testcase for xxx/defaults.go,  we might consider to improve the test coverage for this in the future.

Reply:
defaulting is always required when adding new fields.
we could have better unit testing, yes.